docker/ubuntu/Dockerfile

```diff
@@ -168,12 +196,34 @@ RUN /src/grass_build/configure \
   --with-bzlib \
   --with-pdal \
   --without-mysql \
-  --without-odbc \
-  --without-openmp \
+  --with-blas \
+  --with-lapack \
+  --with-readline \
+  --with-odbc \
+  --with-openmp \
   --without-opengl \
     && make -j $NUMTHREADS \
     && make install && ldconfig
 
+# Build the GDAL-GRASS plugin
+RUN git clone https://github.com/OSGeo/gdal-grass && \
+    cd "gdal-grass" && \
+    ./configure --with-gdal=/usr/bin/gdal-config \
+            --with-grass=/usr/local/grass84 && \
+    make -j $NUMTHREADS && \
+    make install -j $NUMTHREADS && \
+    cd /src && \
+    rm -rf "gdal-grass"
+
+# Reduce the image size - Remove unnecessary grass files
+RUN cp /usr/local/grass84/gui/wxpython/xml/module_items.xml module_items.xml; \
+    rm -rf /usr/local/grass84/demolocation; \
+    rm -rf /usr/local/grass84/fonts; \
+    rm -rf /usr/local/grass84/gui; \
+    rm -rf /usr/local/grass84/share; \
+    mkdir -p /usr/local/grass84/gui/wxpython/xml/; \
+    mv module_items.xml /usr/local/grass84/gui/wxpython/xml/module_items.xml;
+
 # Unset environmental variables to avoid later compilation issues
 ENV INTEL ""
 ENV MYCFLAGS ""
```

Comment:
With the removed lines, there is no other compilation after, so these ENV are now useless, since these ENV don't affect the env when a new "FROM" is used.